lib/Authsignal/AuthsignalClient.php

```diff
@@ -16,7 +16,7 @@ public static function apiUrl($url='')
   public function handleApiError($response, $status)
   {
     $type = $response['error'] ?? null;
-    $msg  = $response['message'];
+    $msg  = $response['message'] ?? $response['errorDescription'] ?? null;
```

Comment:
Hey @vukhuu thanks for submitting a PR.

Our standard error response model looks like:

```
{
  "error": string,
  "errorDescription": string
}
```

We'll look at documenting this

```suggestion
    $msg  = $response['errorDescription'] ?? null;
```

Cheers!